contracts/halo-pool/src/msg.rs

```diff
@@ -41,6 +41,10 @@ pub enum ExecuteMsg {
 pub enum QueryMsg {
     #[returns(PoolInfo)]
     Pool {},
-    // TODO: add query for user's reward balance
-    // TODO: add query for user's staked balance
+    #[returns(RewardTokenAsset)]
+	PendingReward {
+		address: Addr,
```

Comment:
những chỗ input thì để kiểu string sẽ tốt hơn. khi nào vào xử lý thì mình sẽ convert sau

Reply:
updated, thank you